.goreleaser.yml

```diff
@@ -0,0 +1,78 @@
+# This is an example .goreleaser.yml file with some sane defaults.
```

Comment:
Thoughts on pruning configuration that (re)sets to defaults?

Reply:
I have found it re-assuring / valuable when reading this file, to see the different configuration options set (even if they are set to their default).

i.e. seeing `draft: true` in this file. 
If I were to remove this line, I think I would check the docs every time to make sure that the default is to publish a draft.